neuro_san/service/agent_main_loop.py

```diff
@@ -94,6 +97,9 @@ def parse_args(self):
                                 default=os.environ.get("AGENT_OPENAPI_SPEC",
                                                        self.service_openapi_spec_file),
                                 help="File path to OpenAPI service specification document.")
+        arg_parser.add_argument("--manifest_file_update_period", type=int,
+                                default=int(os.environ.get("AGENT_MANIFEST_UPDATE_PERIOD", "0")),
+                                help="Periodic update period for manifest in seconds.")
 
```

Comment:
Adding one more configuration parameter for server:
period for manifest registry update in run time.
Period is specified in seconds, value == 0 (default) will disable updates.

Reply:
Need: 
1) You need units to be in the name of these "update_period" member/env variables throughout. Please add "seconds" (most likely?) for clarity.
2) You should add this env var to the example Dockerfile we ship with a comment as to its semantics
3) In the help text here and in the Dockerfile you should add a blurb about what setting this to <=0 would mean. I suspect it means "Reading updates is not allowed" and that is a reasonable and secure default behavior with default of 0 you have now.

Reply:
OK done.